### PR TITLE
Add configuration necessary to install monasca-agent on CentOS 8

### PR DESCRIPTION
**Pull request by oneswig**
Different method for managing package name dependencies to enable
a distinction between CentOS 8 and CentOS 7 and portable deployment to either.

### CODE REVIEW

File: vars/deps_SUSE.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - python-virtualenv
+  - python-PyYAML
+  - libxml2-devel
+  - libxslt-devel
+  - libffi-devel
+  - openssl-devel
+  - gcc
+  - lshw
+
+...
```
Comment (dougszumski) [inline vars/deps_SUSE.yml]: Nit: remove these?

File: tasks/install.yml [hunk 2]
```diff
@@ -3,20 +3,20 @@
...
-- name: Install rpms to avoid pip doing compilation or enable it
-  yum: name={{ item }} state=present
-  with_items: "{{ dependencies_redhat }}"
-  when: ansible_os_family == 'RedHat'
+- name: Set OS distribution dependent variables
+  include_vars: "{{ lookup('first_found', params) }}"
+  vars:
+    params:
+      files:
+        - "deps_{{ ansible_os_family }}_{{ ansible_distribution_major_version }}.yml"
```
Comment (dougszumski) [inline tasks/install.yml:11]: Much cleaner, thanks. I haven't tested it on the various distros, but hopefully this is enough. They probably need attention anyway.

**Review by dougszumski — CHANGES_REQUESTED** [review]

File: vars/deps_SUSE.yml (hunk 1 above)
Comment (oneswig) [reply vars/deps_SUSE.yml]: It's true I made no effort to test SUSE package installs, but should we address the question of removing SUSE support in a separate PR?

Comment (dougszumski) [reply vars/deps_SUSE.yml]: What's with the `...` though?

Comment (oneswig) [reply vars/deps_SUSE.yml]: I've tended to add the `...` closer syntax if I've added the `---` opener syntax.  Similar to brackets perhaps.  It's optional and I've removed it in the next commit.

Comment (dougszumski) [reply vars/deps_SUSE.yml]: Thanks - I've never seen anyone close their YAML document in an Ansible context. I suppose each file is a document.

**Pull request merged**